docs/tools/glossary.md

```diff
@@ -53,7 +53,7 @@ The Airgrab model is designed to be a more capital efficient way to deploy dApp
 
 * * *
 
-### API
+### API (Application programming interface)
```

Comment:
Think is better "Application Programming Interface" for this title

Reply:
Totally agree.